### DISCUSSION TITLE
Where is the artifact for 'configuring model views' located?

### DISCUSSION THREAD

**Discussion by ShawnMcCool** [strapi/strapi#8509] [Legacy Topics]
I rearranged the fields in my model so that they are in a good order and look good. However i'm not sure where this change exists. I don't see it in the JSON files and I don't see it in the database. 

I'd appreciate it if someone can point me in the right direction. Thanks.

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
